app/views/conversions/projects/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for @project, url: conversions_update_path(@project_id), method: :post do |form| %>
```

Comment:
You can just pass `@project` in `conversions_update_path(@project_id)`, Rails will figure out it needs the ID of the project.

Reply:
This throws an error when I just pass in `@project`.
ID is coming back `nil`.
Says missing required keys: [:id]